websites/E/Emby/dist/metadata.json

```
@@ -40,6 +40,39 @@
 			"title": "Show timestamps",
 			"icon": "fad fa-stopwatch",
 			"value": true
+		},
+		{
+			"id": "showRichImages",
+			"title": "Show rich images",
+			"icon": "fad fa-sparkles",
+			"value": true
```

I would suggest using one setting like the following
```suggestion
			"id": "thumbnail",
			"title": "Show thumbnails",
			"icon": "fad fa-sparkles",
			"value": true
```